frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/parsers/callout_node_parser.dart

```diff
@@ -10,7 +10,6 @@ class CalloutNodeParser extends NodeParser {
   @override
   String transform(Node node, DocumentMarkdownEncoder? encoder) {
     assert(node.children.isEmpty);
-    final icon = node.attributes[CalloutBlockKeys.icon];
```

Comment:
We need to check whether the icon value is an icon type or an emoji type. If it's an emoji type, we should include it in the Markdown.

Reply:
Done!

Reply:
The CI failed due to merge conflicts.

Merge the latest main branch after this PR is merged: https://github.com/AppFlowy-IO/AppFlowy/pull/7491